README.md

```diff
@@ -611,7 +611,6 @@ The HttpClient offers a simplified client HTTP API for Angular applications that
   * Stagehand
     * [google/stagehand](https://github.com/google/stagehand)
 * Scaffold
-  * [Angular Firebase](https://angular-firebase.com): Build a complete Angular Firebase web app in minutes
```

Comment:
You've removed the wrong line here.

Reply:
Hi, the removed line is the correct one. 

Perhaps the commit text is not accurate enough.

The link to Angular Firebase under the topic Scaffold does not exist anymore. 
And as written in the issue both links (the firebase and the scaffoldhub) pointed to the same.
So I decided to go with the working link :-)

Reply:
yeah it doesn't look like it works anymore